doc/todo/RELEASE

```diff
@@ -9,6 +9,59 @@ Following shell expansions are used in this document:
  ~elektra-ftp .. where ftp files go to
  ~elektra-doc .. where doc flies go to
 
+----------------------------------------------------------------------
+
+This describes what to do for the automated release:
+
+= Tasks before Release =
+
+== Updates ==
+
+- Update doc/COMPILE.md to reflect actually tested setups
+- Write release notes
+
+== Increment Version Number ==
+
+- CMakeLists.txt
+- ~e/scripts/dev/build-debian-package
+- Change VERSION variable in build-server
+- Commit and push the changes
+
+== Check ==
+
+- Everything blue on https://build.libelektra.org/
+- Trigger release pipeline with dry run
+- Inspect the logs, artifacts and the website from the pipeline
+  - [ ] error logs (*.error) must be empty
+  - [ ] check in the (test-*) folder if all tests passed
+  - [ ] check if version is correct in the version file
+  - [ ] 
+
+== Preperation ==
+- Finalize release notes (hash sums etc are already known)
+- cd ~e && cp doc/todo/NEWS.md doc/news/_preparation_next_release.md
+
+== Publish ==
+- Trigger release pipeline without dry run
+- (Possible) rebuilds from website for last changes in the release notes
+
+= Announce =
+send out release mails&post on social media #676
+(Personal notice: see also NOTIFY)
+
+
+= TODO =
+- Docker container with sudo rights so that the built deb-packages can be installed and tested in the release pipeline
```

Comment:
It would be better to create a new Docker container where the deb packages are already included (via Dockerfile). Is there some problem doing it that way?

Reply:
I agree.

We should also separate release and development builds of the deb-packages. What we do right now with the debs is quite confusing.